core/src/toga/widgets/canvas.py

```diff
@@ -1445,11 +1451,20 @@ def measure_text(
     # As image
     ###########################################################################
 
-    def as_image(self) -> toga.Image:
+    def as_image(self, format: Any|None=None) -> toga.Image:
+        
         """Render the canvas as an Image.
-
-        :returns: A :class:`toga.Image` containing the canvas content."""
-        return Image(data=self._impl.get_image_data())
+        :param format: None or type of the image that will be returned
+        Supported Types: PIL.Image.Image, toga.Image
+        :returns: Object of the specified Image type. If unspecified returns toga.Image"""
+        if format == None:
+            return Image(data=self._impl.get_image_data())
+        elif isinstance(format, toga.Image):
+            return Image(data=self._impl.get_image_data())
```

Comment:
This effectively means that format has a default value of `toga.Image`. This simplifies the type description, and simplifies the implementation because there's no duplicated code path.